.github/workflows/ark.unit.yaml

```diff
@@ -78,10 +78,7 @@ jobs:
         with:
           version: v1.61
           working-directory: ./pkg/client-sdk
-          config-content: |
-            issues:
-              exclude-files:
-                - '^test/wasm/wasm_test\.go$'
+          args: --timeout 5m --skip-files=.*_test.go
```

Comment:
`--skip-files` not needed, can be dropped?

Reply:
I am having linting errors in wasm_test.go also I was thinking that linting tests doesnt make much sense if tests are passing, we only lose bit time in CI and bit more when inspecting linting errors.